src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -201,23 +201,31 @@ function plant_absorbed_pfd(
     frac_diff::FT,
 ) where {FT}
 
-    (; ld, Ω, n_layers) = RT.parameters
+    (; χl, Ω, n_layers) = RT.parameters
 
+    ϕ1 = 0.5 - 0.633 * χl - 0.33 * χl^2
+    ϕ2 = 0.877 * (1 - 2 * ϕ1)
+    μ = cos(θs)
     # Compute μ̄, the average inverse diffuse optical length per LAI
-    μ̄ = 1 / (2 * ld)
+    μ̄ = 1 / ϕ2 * (1 - (ϕ1 / ϕ2) * log((ϕ1 + ϕ2) / ϕ1))
```

Comment:
I don't understand this part. I think adding ld(χl) still holds:

μ̄ = 1 / (2 * ld)

Reply:
This is a good point, I should try this way as opposed to changing all the equations. When I implemented this I just changed all of the relevant equations to match section 2.3.1 [here](https://escomp.github.io/ctsm-docs/versions/release-clm5.0/html/tech_note/Surface_Albedos/CLM50_Tech_Note_Surface_Albedos.html#canopy-radiative-transfer)